packages/preset-icons/src/cdn.ts

```diff
@@ -2,7 +2,19 @@ import type { IconifyJSON } from '@iconify/types'
 import { loadIcon } from '@iconify/utils/lib/loader/loader'
 import { searchForIcon } from '@iconify/utils/lib/loader/modern'
 import type { UniversalIconLoader } from '@iconify/utils/lib/loader/types'
-import { $fetch } from 'ohmyfetch'
+
+const myFetch = async function myFetch(url: string) {
+  if (typeof window !== 'undefined' && window.fetch) {
+    // browser
+    const res = await window.fetch(url)
+    return await res.json()
```

Comment:
shouldn't check for res and also for status? if the cdn is down/missing or the response is not 20X you will get weird errors:
```ts
return res && res.ok 
  ? await res.json()
  : throw new Error(`unexpected/missing response from ${url}: ${res?.status ?? 'no response'}`)
```
```

Reply:
@userquin I have update it.